### ISSUE TITLE
Working with headers in result

### ISSUE THREAD

**Issue by neurotech** [brianleroux/tiny-json-http#1]
Hi @brianleroux 

Is there any way to "get" the headers of the result returned by `tiny-json-http`? Below is some example code that may help explain my question:

```javascript
tiny.get(options, function (err, result) {
  if (err) {
    console.error(err);
  }
  if (result.headers.link) {
    console.log('Link header present.');
  } else {
    console.log('No Link header present.');
  }
});
```

**Comment by brianleroux** [comment]
I really debated this and decided I'd wait and see if someone wanted it! 1/2

**Comment by brianleroux** [comment]
How do you think it should work? Could see the return object being `{headers, body}` maybe?

**Comment by neurotech** [comment]
That sounds great to me. In my mind's eye I see something like:

```javascript
{
  headers: {
    date: 'foo',
    link: 'baz',
    ...
  },
  body: [...]
}
```

**Comment by brianleroux** [comment]
another way, backwards compatible (tho idc about that hugely as its such a small lib)

```javascript
tiny.get(options, function _got(err, body, headers) {
  console.log({err, body, headers})
})
```

**Comment by neurotech** [comment]
I just tried that, and got this in return:

```javascript
{
  err: null,
  body: [...],
  headers: undefined
}
```

**Comment by brianleroux** [comment]
oh ya! that doesn't exist yet / just brainstorming!! wanna get @ednapiranha's feedback too

**Comment by neurotech** [comment]
Ah my apologies I misunderstood.

**Comment by neurotech** [comment]
@brianleroux do you have a rough idea of how long it will take to get this implemented? (Just planning ahead with regards to a Christmas project). Please let me know as soon as you can.

**Comment by brianleroux** [comment]
oh sorry / I'll gotcha something today promise! 💯

**Comment by brianleroux** [comment]
Aight @neurotech just published 🌟 shiny `5.0`with support for `{body, headers}` in the callback continuation result. Hope this helps!

**Comment by neurotech** [comment]
Thank you so much, you are top shelf! ✨
